gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1312,6 +1312,18 @@ public void xattr_statistics() throws IOException {
     assertThat(myGhfs.delete(testRoot, true)).isTrue();
   }
 
+  public void http_IOstatistics() throws IOException {
```

Comment:
Add the following tests
1. Test counters for HEAD requests
2. Test failure counters -- use unit tests
3. Test counters are not incremented when seeked within inplaceseeklimit 
              ref to test `seekUnderInplaceSeekLimitReadsCorrectBufferedData` for example. Assert that the stats after first and second read calls are same


We need to add tests in both gcs and gcsio layer.